internal/processmon/processmon.go

```diff
@@ -38,107 +33,117 @@ var (
 	ErrFailedtoLaunch = errors.New("Failed to launch enforcer")
 	// ErrProcessDoesNotExists Exported
 	ErrProcessDoesNotExists = errors.New("Process in that context does not exist")
-	//ErrBinaryNotFound Exported
+	// ErrBinaryNotFound Exported
 	ErrBinaryNotFound = errors.New("Enforcer Binary not found")
 )
 
 const (
+	// netnspath holds the directory to ensure ip netns command works
+	netnspath               = "/var/run/netns/"
 	processMonitorCacheName = "ProcessMonitorCache"
 	secretLength            = 32
 )
 
-//ProcessMon exported
-type ProcessMon struct {
+// processMon is an instance of processMonitor
+type processMon struct {
```

Comment:
comments on private structure are not really needed until they tell something really important